tor_core/config.py

```diff
@@ -175,3 +178,6 @@ class Config:
 
 # section for gifs
 config.no_gifs = []
+
+# enables debug information for the cherrypy heartbeat server
+config.heartbeat_logging = False
```

Comment:
I never thought of giving the option to log or not just for the heartbeat server. Good idea! 👍